script.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line max-classes-per-file
+class Book {
+  constructor(title, author) {
```

Comment:
- [ ] Kindly remove these two eslint-disable statements from this file:

- ` /* eslint-disable class-methods-use-this */`
- `/* eslint-disable no-unused-vars */`

The only eslint-disable statement allowed for this milestone is that of the `// eslint-disable-next-line max-classes-per-file` because you have not been introduced to the JavaScript modules. 

`Eslint` helps developers write clean, readable and industry-standard JavaScript codes and using `eslint-disable` statements defeats this purpose. Kindly implement this suggestion. 👍 😎